.env.example

```
@@ -1,5 +1,24 @@
-PORT=3333
-HOST=0.0.0.0
-NODE_ENV=development
-APP_KEY=73rHnSEgVPLfFLpTk8omwzBrMk-c75rX
-DRIVE_DISK=local
+# Application Environment Variables
+NODE_ENV=
+PORT=
+API_URL=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)